webapp/src/App.tsx

```diff
@@ -129,16 +129,18 @@ const App: FC = () => {
                 <div className={classes.container}>
                     <div className={classes.header}>
                         <Subtitle1 as="h1">Chat Copilot</Subtitle1>
-                        <div className={classes.cornerItems}>
-                            <div data-testid="logOutMenuList" className={classes.cornerItems}>
-                                <PluginGallery />
-                                <UserSettingsMenu
-                                    setLoadingState={() => {
-                                        setAppState(AppState.SigningOut);
-                                    }}
-                                />
+                        {appState > AppState.SettingUserInfo && (
```

Comment:
intentional?

Reply:
should the check be `!==` then? right now, the items will only show when `appState == AppState.SigningOut`.